src/lib/bigNums.js

```diff
@@ -1,5 +1,5 @@
 "use strict";
-const Big = require("big.js");
+const Big = require("../lib/fixedDecimal.js");
 const BigAcd = Big(); // ACD values
```

Comment:
nitpicking for future readability : could we easily do a search & replace for BigAcd -->  FixedAcd , BigEth -> FixedEth , BigPt -> FixedPt in the whole codebase?